roles/tendrl-api/tasks/binary.yml

```diff
@@ -0,0 +1,53 @@
+- name: Install tendrl-api dependencies
+  yum:
+    name={{ item }}
+    state=present
+    validate_certs=no
+  with_items:
+    - epel-release
```

Comment:
Where does this `epel-release` package come from? We have a role for epel already.